### ISSUE TITLE
Sceneform particle system?

### ISSUE THREAD

**Issue by szczepanskikrs** [google-ar/sceneform-android-sdk#597]
Hello, first of all, thanks for a great update with FBX animations in Sceneform 1.7.0. I have a feature request, do you guys plan to implement some particle system into Sceneform?  Both ViroCoreAR and ARKit have one...

**Comment by 544531645** [comment]
Can your animation work properly? I was able to work in the virocore FBX file before. Now the model will disappear when the animation starts on sceneform

**Comment by 544531645** [comment]
Some FBX transformation SFB files also fail to create sfa files

**Comment by szczepanskikrs** [comment]
Yes, the work, the FBX file needs to be valid, if you have animation files in a separate file you need to add them when importing. Also if you import assets with textures remember that you need to have relative paths to textures. Also, the roughness map needs to only use the red channel if you use one. I haven't figured out how to create valid roughness map yet since Sceneform docs aren't super precise on this part...

**Comment by 544531645** [comment]
My animation is included in the FBX file, similar to the dae file. Could you please help me check my file? Thank you
[apple.zip](https://github.com/google-ar/sceneform-android-sdk/files/2970477/apple.zip)

**Comment by 544531645** [comment]
This FBX file works in both virocore and web, but cannot be converted to an SFB file
